### ISSUE TITLE
Build from source fails on OS X: OpenMP requirement and warnings as errors

### ISSUE THREAD

**Issue by buhrmann** [src-d/wmd-relax#6]
Building from source on OS X (at least in my environment) fails due to the OpenMP requirement in CMakeLists.txt. After removing this requirement the compile starts but fails after several warnings are treated as errors (-Werror):

```
/Users/thomas/code/wmd-relax/or-tools/src/base/sysinfo.cc:35:10: warning: unused variable 'task' [-Wunused-variable]
  task_t task = MACH_PORT_NULL;
         ^
1 warning generated.
In file included from /Users/thomas/code/wmd-relax/python.cc:7:
/Users/thomas/code/wmd-relax/emd.h:91:9: warning: using integer absolute value function 'abs' when argument is of
      floating point type [-Wabsolute-value]
    if (abs(sum - MASS_MULT + 0.) / MASS_MULT > 0.0000001) {
        ^
/Users/thomas/code/wmd-relax/emd.h:91:9: note: use function 'std::abs' instead
    if (abs(sum - MASS_MULT + 0.) / MASS_MULT > 0.0000001) {
        ^~~
        std::abs
/Users/thomas/code/wmd-relax/emd.h:91:9: warning: using integer absolute value function 'abs' when argument is of
      floating point type [-Wabsolute-value]
    if (abs(sum - MASS_MULT + 0.) / MASS_MULT > 0.0000001) {
        ^
/Users/thomas/code/wmd-relax/emd.h:150:3: note: in instantiation of function template specialization '(anonymous
      namespace)::convert_weights<float>' requested here
  convert_weights(w1, false, demand, size);
  ^
/Users/thomas/code/wmd-relax/python.cc:201:40: note: in instantiation of function template specialization 'emd<float>'
      requested here
  return emd_entry(self, args, kwargs, emd<float>);
                                       ^
/Users/thomas/code/wmd-relax/emd.h:91:9: note: use function 'std::abs' instead
    if (abs(sum - MASS_MULT + 0.) / MASS_MULT > 0.0000001) {
        ^~~
        std::abs
/Users/thomas/code/wmd-relax/emd.h:96:15: warning: format specifies type 'long' but the argument has type 'int64_t'
      (aka 'long long') [-Wformat]
              sum, MASS_MULT);
              ^~~
/Users/thomas/code/wmd-relax/emd.h:96:20: warning: format specifies type 'long' but the argument has type 'int64_t'
      (aka 'long long') [-Wformat]
              sum, MASS_MULT);
                   ^~~~~~~~~
4 warnings generated.

```

After also removing the -Werror flag the source "successfully" compiles.

**Comment by vmarkovtsev** [comment]
Clang is weird... I have not tested on macOS yet, will check, thanks.

**Comment by vmarkovtsev** [comment]
Fixed by e52af6a2b35c0010ae74e3a9f1be120bc061c66f
